base/prometheus/prometheus.ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    category: rbac
+    deploy: sourcegraph
+  name: prometheus
+roleRef:
+  apiGroup: ""
+  kind: ClusterRole
+  name: prometheus
+subjects:
+- kind: ServiceAccount
+  name: prometheus
+  namespace: default
```

Comment:
My comment about making sure that the namespace matches up also applies here

Reply:
done